controller/controller.go

```diff
@@ -0,0 +1,46 @@
+package controller

```

Comment:
rename controllers to handlers,that way, it's easier for other Golang developers to understand your code